### ISSUE TITLE
Remove dependency of deprecated "rollup-plugin-terser": "^7.0.0"

### ISSUE THREAD

**Issue by sushmakale2016** [GoogleChrome/workbox#3200]
Please remove dependency of "rollup-plugin-terser": "^7.0.0" plugin, as it causes warning inside the dependent applications

**Comment by MohdFaisalBidda** [comment]
I want to work on it can you please assign me this issue @jeffposnick

**Comment by sushmakale2016** [comment]
I am not able to assign the issue to anyone field is not editable for me

**Comment by lucasff** [comment]
Hello, I'm also facing this issue, due to this package being a dependency of react-scripts! Everyone that uses CRA will face the same.

Edit:

```
"overrides": {
  "rollup-plugin-terser": "@rollup/plugin-terser@0.4.1"
},
```

**Comment by drewbitt** [comment]
Awaiting #3170

**Comment by tannerstern** [comment]
It would appear that this will be resolved shortly, as all reviewers have approved #3170. We've only recently started seeing these "deprecation" warnings ourselves.

**Comment by pwbriggs** [comment]
Fixed in #3170.

**Comment by tomayac** [comment]
Hi there,

Workbox is [moving to a new engineering team within Google](https://github.com/GoogleChrome/workbox?tab=readme-ov-file#maintenance-update:~:text=Maintenance%20update,owners%20of%20Workbox.). As part of this move, we're declaring a partial bug bankruptcy to allow the new team to start fresh. We realize this isn't optimal, but realistically, this is the only way we see it working. For transparency, here're the criteria we applied:

- For PRs, we closed [everything that doesn't target the current `v7` branch](https://github.com/GoogleChrome/workbox/pulls?q=is%3Apr+is%3Aclosed+-base%3Av7).
- For Issues, we closed [everything that was created before 2024](https://github.com/GoogleChrome/workbox/issues?q=is%3Aissue+is%3Aclosed+created%3A%3C2024-01-01+). 

Thanks, and we hope for your understanding!
The Workbox team
